v8.rb

```diff
@@ -18,7 +18,7 @@ class V8 < Formula
   # https://github.com/Homebrew/homebrew/issues/21426
   depends_on :macos => :lion
 
-  depends_on :python => :build
+  depends_on python => :build
```

Comment:
I came here because I saw the same warning – thanks for your efforts!

The warning includes double quotes around `python` to make the whole line `depends_on "python" => :build` (similar to lines below).

Reply:
thanks, fixed!